agent/common.sh

```diff
@@ -49,3 +49,4 @@ function getReleaseImage() {
 export MACHINE_CONFIG_SERVER_PORT=22623
 export KUBE_API_PORT=6443
 export INGRESS_ROUTER_PORT=443
+export AGENT_NODE0_IPSV6=${AGENT_NODE0_IPSV6:-}
```

Comment:
This new var is added to be able to correctly cleanup the entries in agent/cleanup.sh. It is not directly accessible otherwise. ```./agent/cleanup.sh: line 37: AGENT_NODES_IPSV6[0]: unbound variable```

Reply:
Moreover, these DNS entries resolutions are only needed in case of SNO IPV6 configuration only to run the openshift conformance tests.